src/routers/TabRouter.js

```diff
@@ -61,11 +61,9 @@ export default (
         const routes = order.map((routeName: string) => {
           const tabRouter = tabRouters[routeName];
           if (tabRouter) {
-            const childAction =
-              action.action ||
-              NavigationActions.init({
-                ...(action.params ? { params: action.params } : {}),
-              });
+            const childAction = NavigationActions.init({
+              ...(action.params ? { params: action.params } : {}),
```

Comment:
I also wonder if we should ever pass `action.params` in here... we’re just supposed to be initializing a blank navigation state, right? That should have nothing to do with the action we were passed.

Reply:
Can you actually go ahead and remove `action.params` here? (just pass an empty object)

Reply:
I'm not totally sure why this is here, but removing params here breaks a test (TabRouter › Handles passing params to nested tabs)

Reply:
I think the test case is broken. #3094